README.md

```diff
@@ -202,6 +204,19 @@ var movies = await this.index.Search<Movie>("ironman");
 var movies = await this.index.Search<Movie>("ironman", new SearchQuery {Limit = 100});
 ```
 
+## 🧰 Use a Custom HTTP Client
+
+You can replace the default client used in this package by the one you want.
+
+For example:
+
+```c#
+var _httpClient = ClientFactory.Instance.CreateClient<MeilisearchClient>();
```

Comment:
This example might need an update. The way we have created Meilisearch , It can be used as a Typed HttpClient. More details here: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-3.1#typed-clients

Reply:
Hum I'm not sure to really understand. Can you write down the example you would like to see in the README instead of this one? 🙂

Reply:
Sure will do